lib/config/types.ts

```diff
@@ -357,6 +357,12 @@ export interface RenovateOptionBase {
   experimentalDescription?: string;
 
   experimentalIssues?: number[];
+
+  advancedUse?: boolean;
+
+  advancedFunctionality?: boolean;
+
+  advancedConfig?: boolean;
```

Comment:
can we combine these?

Reply:
yes we can if we will use the same string, but if we need 3 different strings I should check the `option name`, and according to this, we can use the relevant string.
something like this:
```
      if (el.advancedUse) {
        if (option.name === 'excludeCommitPaths') {
          configOptionsRaw[headerIndex] += generateAdvancedUse();
        } 
        if(option.name ==='commitMessagePrefix'||option.name ==='commitMessageAction'||
          option.name ==='commitMessageTopic'||option.name ==='commitMessageExtra'||
          option.name ==='commitMessageSuffix'){
          configOptionsRaw[footerIndex] += generateAdvancedUse();
        }
        if(option.name === 'group'|| option.name === 'followTag'){
          configOptionsRaw[headerIndex] += generateAdvancedFunctionality();
        }
        if(option.name === 'insecureRegistry'){
          configOptionsRaw[footerIndex] += generateAdvancedConfig();
        }
      }
```

Reply:
I think we can use a simple common admonition text without such complicated logic. If more details are needed for individual options then it can be stated in the markdown documentation separately

Reply:
That's what i suggested too before 😕

Reply:
sounds perfect, what do you recommend for the common text?

Reply:
```
<!-- prettier-ignore -->
!!! warning
For advanced use only! Use at your own risk!
```

can we use this text for all the options?